app/Actions/Network/RemoveServerFromNetwork.php

```
@@ -0,0 +1,37 @@
+
+        if ($network->type === NetworkType::WIREGUARD) {
+            $network->load('servers.server');
+            foreach ($network->servers as $sibling) {
+                if ($sibling->id !== $member->id
+                    && in_array($sibling->status, [NetworkServerStatus::ACTIVE, NetworkServerStatus::UPDATING], true)) {
+                    $this->sync->toPresent($sibling);
+                }
+            }
+        } else {
+            $this->firewall->handle($network);
+        }
```

PHPMD (2.15.0)

[error] The method remove uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)